homeassistant/components/recorder/db_schema.py

```diff
@@ -513,9 +513,7 @@ class StatisticsBase:
     """Statistics base class."""
 
     id: Mapped[int] = mapped_column(Integer, Identity(), primary_key=True)
-    created: Mapped[datetime] = mapped_column(
-        DATETIME_TYPE, default=dt_util.utcnow
-    )  # No longer used
+    created: Mapped[datetime | None] = mapped_column(DATETIME_TYPE)  # No longer used
     created_ts: Mapped[float] = mapped_column(TIMESTAMP_TYPE, default=time.time)
```

Comment:
This needs to allow `None` like the original pre sqlalchemy 2.0 schema

https://github.com/home-assistant/core/blob/a7e42798da18b9e46a824dc85a91eebb30e39192/homeassistant/components/recorder/db_schema.py#L464